tests/unit/test-cmd/test.run.js

```diff
@@ -477,6 +478,18 @@ describe('run', () => {
         });
     });
 
+    it('pauses the web-ext process (CTRL+Z in shell console)', () => {
+      const {reloadStrategy, kill} = prepare();
+      const fakeStdin = new tty.ReadStream();
+
+      return reloadStrategy({}, {stdin: fakeStdin})
+        .then(() => {
+          fakeStdin.emit('keypress', 'z', {name: 'z', ctrl: true});
+        }).then(() => {
+          assert.ok(kill.called);
```

Comment:
@saintsebastian I've been thinking about how to test this behavior in the unit tests and I think that we can:

- assert that kill is called with the expected pid and the expected signal
- assert that `stdin.setRawMode(false)` has been called on `fakeStdin` **before** the call to the `kill` sinon spy
- assert that `stdin.setRawMode(true)` has been called on `fakeStdin` **after** the call to the `kill` sinon spy

The 3 assertions above should be enough to ensure that the real (non mocked) behavior would be the expected one.

How that sounds to you?

(side note: we should also make sure that we are going to exit the `while` loop related to the keypress processing, https://github.com/saintsebastian/web-ext/blob/c6d78220df21e4f37779643468a8d9bb3876561d/src/cmd/run.js#L156-L186, when we exit these tests)